### PR TITLE
Add hf:// as an alias to huggingface://

### PR DESCRIPTION
**Pull request by ericcurtin**
Also check start of string contains :// , by checking if start of the string simply starts with things like "hf", "ollama", etc. we are kinda polluting the namespace and not allowing models to start with these terms.

### CODE REVIEW

**Comment by ericcurtin** [comment]
Haven't run tests locally, must keep an extra eye on the builds

File: ramalama/common.py [hunk 1]
```diff
@@ -69,7 +69,11 @@ def run_cmd(args, cwd=None, ignore_stderr=False):
     if ignore_stderr:
         stderr=subprocess.PIPE
 
-    return subprocess.run(args, check=True, cwd=cwd, stdout=subprocess.PIPE, stderr=stderr)
+    try:
+        return subprocess.run(args, check=True, cwd=cwd, stdout=subprocess.PIPE, stderr=stderr)
+    except Exception:
+        perror(f"subprocess.run({args}, check=True, cwd={cwd}, stdout=subprocess.PIPE, stderr=stderr)")
```
Comment (rhatdan) [inline ramalama/common.py:75]: This line here is for debugging?  It is causing tests to fail. The docker error is caught above and ignored.

Comment (ericcurtin) [reply ramalama/common.py:75]: I'll remove it, it was for debugging, I was hoping to keep it in, but will remove for now

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**